docs/BasicCommunication/OnAppRegistered/index.md

```diff
@@ -48,7 +48,9 @@ If the application does NOT resume data successfully:
   3. Compile and store `VRGrammars` for the `vrSynonyms` parameter, and arrange them for the user to be able to use via voice recognition. Note: The VR commands to activate an application must be accessible when viewing a different active application or the list of registered applications.
   4. Provide the user with the possibility to choose an application among a list of registered applications.
   5. Send an `OnAppActivated` notification to SDL when the user activates an app via the `UI` or `VR`.   
-  6. Manage application events by priority. HMI gets proirity information from _OnAppRegistered_, _UpdateAppList_, _ActivateApp_ HMI API.
+  6. Manage application events by priority. HMI gets proirity information from _OnAppRegistered_, _UpdateAppList_, _ActivateApp_ HMI API.  
+  7. HMI must set app icon and create the app title in the mobile apps list in case SDL provides `<icon>` via `OnAppRegistered` notification.  
+  8. HMI must set default AppLink app icon in case SDL omits `<icon>` at `OnAppRegistered` notification.
```

Comment:
Avoid `AppLink` language, this term can probably just be omitted here.

Reply:
fixed https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/95/commits/a83706c3263d276107be618b589895dce7caf8f3